src/models/blocks/view.rs

```diff
@@ -88,6 +88,7 @@ pub struct SlackViewStateValue {
     pub value: Option<String>,
     pub selected_date: Option<String>,
     pub selected_time: Option<String>,
+    pub selected_date_time: Option<i64>,
```

Comment:
If this is UNIX ts in seconds I think we have this type already in some other places:
> SlackDateTime

Reply:
Oh, cool! Missed that. Will fix it.